functions/log/2.0/index.js

```diff
@@ -0,0 +1,15 @@
+const log = async ({ severity, variables }) => {
+  const severityTypes = ['log', 'info', 'debug', 'warn', 'error'];
```

Comment:
There is no severity "log", so we should leave this out

Reply:
I got the info from : [https://github.com/bettyblocks/cli/wiki/Functions:Logging](https://github.com/bettyblocks/cli/wiki/Functions:Logging)

Reply:
Correct, though there is no `log` logging severity. There is a `console.log()` function but no loglevel / severity